doc/source/new-project-guide/level-3-tutorial.rst

```diff
@@ -48,22 +51,109 @@ When you ``cd`` into the new directory, you will see a folder structure as shown
      ├── proj-one
      │   ├── __init__.py
      │   └── proj_one_code.py
-     ├── proj-two
-     │   ├── __init__.py
-     │   └── proj_two_code.py
      └── tests
           ├── __init__.py
           └── test_shared_functions.py
 
-Please take a look at those files created with your favorite IDE software (e.g., Visual Studio Code, PyCharm, etc.).
+If you would like to add more projects, you can create a new folder under the root directory. For example, you can create a folder called ``proj-two`` and add your code there. This can be extrapolated for as many projects as you like:
 
-See the descriptions below for each file created in the project:
+.. code-block:: bash
+
+     mkdir proj-two
+     cd proj-two
+     touch __init__.py
+     touch proj_two_code.py
+
+
+File descriptions
+^^^^^^^^^^^^^^^^^^
+
+Now that the folder structure is created, let's go through the files and folders in the project:
+
+- ``README.md`` is a markdown file for project documentation. Typically, it includes a project description, installation instructions, and usage examples. You can edit this file to add your own project information.
+
+
+- ``shared_functions.py`` is where you define reused code. It includes the following example function to help demonstrate the concept of shared functions. In your project, this is where you would add the code you want to reuse:
+
+.. code-block:: python
+
+     # shared_functions.py
+     import numpy as np
+
+
+     def dot_product(a, b):
+          """Calculate the dot product of two vectors."""
+          return np.dot(a, b)
+
+- ``proj_one/proj_one_code.py`` shows an example of how to import and use the ``dot_product()`` function from ``shared_functions.py``:
+
+.. code-block:: python
+
+     # proj_one_code.py
+     import shared_functions
+
+     a = [1, 2, 3]
+     b = [1, 2, 3]
+     result = shared_functions.dot_product(a, b)
+     print(result)
+
+- ``__init__.py`` files mark directories as Python modules. These are empty by default at this level.
+
+- ``requirements.txt`` lists your project's dependencies, which in this case includes ``numpy`` and ``pytest``:
+
+.. code-block:: python
+
+     # requirements.txt
+     numpy
+     pytest
+
+- The ``tests`` folder contains tests for your shared functions. For example, ``test_shared_functions.py`` includes tests for the ``dot_product`` function using ``pytest``. General naming convention for test files is ``test_<module_name>.py``. Similarly, test function names should be ``test_<function_name>()``. This is how ``pytest`` recognizes them as test functions:
```

Comment:
it can also contain tests for any of the other sub-project modules too.  Maybe add a few words to help new users extend this when they write their own differently named modules elsewhere in the project.  'for example, if you define a function named somthing in a module in shared_functions directory you would create a new file here called whatever and it would contain a test called whichever....